images/opentelemetry/rootfs/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Copyright 2021 The Kubernetes Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+FROM alpine:3.14.2 as builder
+
+COPY . /
+
+RUN apk update \
+	&& apk upgrade \
+	&& apk add -U bash \
+	&& /build.sh
+
+FROM busybox:latest 
+
+COPY --from=builder init_module.sh /usr/local/bin/init_module.sh
+COPY --from=builder /etc/nginx/ /etc/nginx/
+COPY --from=builder /usr/local/lib/ /usr/local/lib
+COPY --from=builder /usr/local/include/opentelemetry /usr/local/include/opentelemetry
+COPY --from=builder /usr/local/include/nlohmann /usr/local/include/nlohmann
```

Comment:
You don't need any of those :)

As you are compiling the module with embedded OTEL things, apparently those are not required. Take a look:

```
ldd otel_ngx_module.so
	linux-vdso.so.1 (0x00007ffe6e3b0000)
	libstdc++.so.6 => /usr/lib/x86_64-linux-gnu/libstdc++.so.6 (0x00007fc1fb1d4000)
	libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007fc1fb1ba000)
	libc.musl-x86_64.so.1 => not found # I was testing in Debian, so ignore it
	libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007fc1fb076000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007fc1faeb1000)
	/lib64/ld-linux-x86-64.so.2 (0x00007fc1fb436000)
```